wallet/wallet.go

```diff
@@ -3809,6 +3819,8 @@ func (w *Wallet) publishTransaction(tx *wire.MsgTx) (*chainhash.Hash, error) {
 	} else {
 		log.Infof("Removed invalid transaction: %v",
 			spew.Sdump(tx))
+		log.Infof("Removed invalid transaction: %x",
```

Comment:
Could the above `spew` be removed now?

Reply:
I think that's nice and useful to have as well since it allows you to quickly see more details about the TX. And the hex is useful if you want to manually re-broadcast for whatever reason.